### PR TITLE
Upgrade to v2.2.0 of the AzureRM Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: resources.go [hunk 1]
```diff
@@ -1228,11 +1236,14 @@ func Provider() tfbridge.ProviderInfo {
 			// IoT Central
 			"azurerm_iotcentral_application": {Tok: azureResource(azureIotCentral, "Application")},
 
-			// HPC
+			//
```
Comment (mikhailshilkov) [inline resources.go]: Is this intentional?

File: resources.go [hunk 2]
```diff
@@ -797,6 +799,11 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_cosmosdb_gremlin_database":   {Tok: azureResource(azureCosmosDB, "GremlinDatabase")},
 			"azurerm_cosmosdb_gremlin_graph":      {Tok: azureResource(azureCosmosDB, "GremlinGraph")},
 
+			// Cost Management
+			"azurerm_cost_management_export_resource_group": {
+				Tok: azureResource(azureCostManagement, "ExportResourceGroup"),
```
Comment (mikhailshilkov) [inline resources.go]: `ExportResourceGroup` sounds a bit weird... Maybe `ResourceGroupExport`?

File: resources.go (hunk 1 above)
Comment (stack72) [reply resources.go]: nope - fixed

File: resources.go (hunk 2 above)
Comment (stack72) [reply resources.go]: fixed :)

**Pull request merged**